examples/docker-build/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "docker-build",
+  "version": "1.0.0",
+  "description": "My wonderful Nuxt.js project",
+  "author": "Utkarsh",
+  "private": true,
+  "scripts": {
+    "dev": "nuxt",
+    "build": "nuxt build",
+    "start": "nuxt start",
+    "generate": "nuxt generate"
+  },
+  "dependencies": {
+    "cross-env": "^5.2.0",
+    "nuxt": "^2.4.0"
```

Comment:
It would be a better idea to advice using `nuxt-start` in `dependencies` and `nuxt` in `devDependencies`